easybuild/easyconfigs/s/supermagic/supermagic-20130104-goolf-1.5.14.eb

```diff
@@ -0,0 +1,27 @@
+# This is an easyconfig file for EasyBuild, see https://github.com/hpcugent/easybuild
+# Authors::   Jordi Blasco <jordi.blasco@nesi.org.nz>
+# License::   New BSD
+
+name = 'supermagic'
+version = '20130104'
+
+homepage = 'https://github.com/hpc/supermagic'
+description = """Very simple MPI sanity code. Nothing more, nothing less."""
+
+toolchain = {'name': 'goolf', 'version': '1.5.14'}
+toolchainopts = {'usempi': True}
+
+git_commit_id = '943a7f4703'
+sources = ['%s.tar.gz' % git_commit_id]
+source_urls = ['https://github.com/hpc/supermagic/archive']
+
+preconfigopts = "./autogen && "
```

Comment:
Load the autotools bundle as a builddep:
builddependencies = [('Autotools', 'XXX')]